transifex/utils.py

```diff
@@ -271,53 +271,57 @@ def merge_pr(self):
 
         retries = 0
         while retries <= self.MAX_MERGE_RETRIES:
-            try:
-                self.pr.merge(merge_method=self.merge_method)
-                logger.info('Merged [%s/#%d].', self.name, self.pr.number)
-                return True
-            except github.GithubException as e:
-                # Assumes only one commit is present on the PR.
-                statuses = self.pr.get_commits()[0].get_statuses()
-
-                # Check for any failing Travis builds. If any are found, notify the team
-                # and move on.
-                if any('travis' in s.context and s.state == 'failure' for s in statuses):
+            # Assumes only one commit is present on the PR.
+            statuses = self.pr.get_commits()[0].get_statuses()
+            for s in statuses:
+                logger.info('[%s] status has status [%s]', s, s.state)
```

Comment:
Nit: you probably only need: `'[%s] has status [%s]'`

Reply:
sorry this was only for testing purposes removed this condition.